db/default/users.rb

```diff
@@ -34,7 +34,7 @@ def prompt_for_admin_email
 end
 
 def create_admin_user
-  if ENV['AUTO_ACCEPT']
+  if ENV.fetch("AUTO_ACCEPT", true)
```

Comment:
Unless we explicitly set this to `false` in ofn-install, any username and password currently supplied via ENV vars will be ignored?

Reply:
Good catch @Matt-Yorkley! That could have been bad, production server with public admin account. What do you think about my new commit?

I would like to rewrite the whole thing but it's not a priority right now with S1 bugs around. So I tried to fix it as quickly as possible.

Reply:
> What do you think about my new commit?

Perfect :ok_hand: 

> ...it's not a priority right now with S1 bugs around. So I tried to fix it as quickly as possible.

Agreed :+1: